stag-library-compiler/src/main/java/com/vimeo/stag/processor/utils/FileGenUtils.java

```diff
@@ -116,4 +122,45 @@ static void writeToResource(@NotNull Filer filer, @NotNull String resourceName,
         }
     }
 
+    /**
+     * Safely closes a closeable.
+     *
+     * @param closeable object to close.
+     */
+    static void close(@Nullable Closeable closeable) {
+        if (closeable == null) {
+            return;
+        }
+        try {
+            closeable.close();
+        } catch (IOException e) {
+            // ignored
+        }
+    }
+
+    /**
+     * Takes a String input and sanitizes it for use
+     * in the {@link CodeBlock} class.
+     *
+     * @param string the string to sanitize.
+     * @return a String safe to use in a {@link CodeBlock}
+     */
+    @NotNull
+    public static String sanitizeCode(@NotNull String string) {
+        return string.replace("$", "$$");
+    }
+
+    /**
+     * Takes a String input that was sanitized for
+     * use in the {@link CodeBlock} class and
+     * desanitizes it for normal use.
+     *
+     * @param string the String to desanitize,
+     * @return a String safe for normal use.
+     */
+    @NotNull
+    public static String desanitizeCode(@NotNull String string) {
+        return string.replace("$$", "$");
```

Comment:
Can you break these strings out into constants? Maybe add a code fold and include the constants and these two methods in it.